src/uncss.js

```diff
@@ -218,18 +218,12 @@ function init(files, options, callback) {
         timeout: 0,
         report: false,
         ignoreSheets: [],
+        html: files,
         // gulp-uncss parameters:
         raw: null
     });
 
-    return promise
-        .using(phantom.init(options.phantom), function () {
-            return getHTML(files, options)
-                .spread(getStylesheets)
-                .spread(getCSS)
-                .spread(processWithTextApi);
-        })
-        .asCallback(callback, { spread: true });
+    serializedQueue.push(options, callback);

```

Comment:
The error was a lint error (needed to make my returns from the function consistent). No one ever uses the return value of this function (they all rely on the passed-in callback being called), so I just made it empty return. But now eslint passes properly.